tests/e2e/cucumber/features/smoke/link.feature

```diff
@@ -65,27 +65,55 @@ Feature: link
     And "Alice" logs out
 
 
-  Scenario: public link for folder (by authenticated user)
+  Scenario: public link for folder and file (by authenticated user)
     Given "Admin" creates following user using API
       | id    |
       | Brian |
     And "Alice" logs in
     And "Alice" creates the following resources
       | resource     | type   |
       | folderPublic | folder |
+    And "Alice" creates the following resources
+      | resource                      | type    | content   |
+      | folderPublic/shareToBrian.txt | txtFile | some text |
+      | folderPublic/shareToBrian.md  | mdFile  | readme    |
+    And "Alice" uploads the following resources via drag-n-drop
+      | resource       |
+      | simple.pdf     |
+      | testavatar.jpg |
     And "Alice" shares the following resources using the sidebar panel
-      | resource     | recipient | type | role     |
-      | folderPublic | Brian     | user | Can edit |
+      | resource       | recipient | type | role     |
+      | folderPublic   | Brian     | user | Can edit |
+      | simple.pdf     | Brian     | user | Can edit |
+      | testavatar.jpg | Brian     | user | Can edit |
     And "Alice" creates a public link for the resource "folderPublic" using the sidebar panel
     And "Alice" renames the most recently created public link of resource "folderPublic" to "folderLink"
+    And "Alice" creates a public link for the resource "folderPublic/shareToBrian.txt" using the sidebar panel
+    And "Alice" renames the most recently created public link of resource "folderPublic/shareToBrian.txt" to "t1"
```

Comment:
Can we change the link name to something meaningful rather than random letters. `link1` `link2` would also do.
```suggestion
    And "Alice" renames the most recently created public link of resource "folderPublic/shareToBrian.txt" to "link1"
```